webapp/src/components/onboarding/Tour.tsx

```diff
@@ -26,20 +26,22 @@ export const Tour = (props: TourProps) => {
     }
 
     const data = () => ({
-        title: tourSteps[currentStep].title,
+        title: steps[currentStep].title,
         stepDuration: getStepDuration(),
         tourDuration: getTourDuration(),
         step: currentStep + 1,
-        totalSteps: tourSteps.length
+        totalSteps: steps.length
     });
 
     const onNext = () => {
         pxt.tickEvent("tour.next", data());
+        const nextStep = currentStep + 1;
```

Comment:
Are `nextStep` (and `prevStep` below) unused?

Reply:
Yup, not even sure why I added them. Maybe from previous iteration I undid, maybe an overzealous ai autocomplete I didn't notice 🤔 Either way, I'll remove.